recipe/cache.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Deployer;
+
+use Deployer\Utility\Httpie;
+
+desc('Reset the opcache using a file strategy');
+task('sumo:opcache:reset-file', function () {
+    $opcacheResetScript = 'opcache_reset.php';
+    $scriptPath = '{{ release_path }}/public/' . $opcacheResetScript;
+
+    run('echo "<?php opcache_reset();" > ' . $scriptPath);
```

Comment:
You should also clear the stat cache, see https://github.com/sumocoders/sumodev_deploy/blob/master/lib/sumodev_deploy/tasks/opcache.rb#L14

Reply:
Fixed